### PR TITLE
[enterprise-4.12] OSDOCS-4136: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4136

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-3887-RN/release_notes/ocp-4-12-release-notes.html#ocp-4-12-switch-aws-load-balancer

QE review:
- [x] QE has approved this change.

Additional information: RN entry for #50439

### CODE REVIEW

**Comment by jldohmann** [comment]
72f80e2..a7eb76d resolve conflicts and rebase

**Comment by jldohmann** [comment]
@lihongan could you PTAL for QE ack? This is the RN entry for [NE-865](https://issues.redhat.com/browse/NE-865). Thank you

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -109,6 +109,13 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 
 For more information, see the xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[supported devices].
 
+[id="ocp-4-12-switch-aws-load-balancer"]
+==== Switch between AWS load balancer types without deleting the Ingress Controller
+
+You can update the Ingress Controller to switch between an AWS Classic Load Balancer (CLB) and an AWS Network Load Balancer (NLB) without deleting the Ingress Controller. Previously, the Ingress Controller had to be force replaced which resulted in an expected outage that could last for several minutes.
```
Comment (gaurav-nelson) [inline release_notes/ocp-4-12-release-notes.adoc:115]:
```suggestion
You can update the Ingress Controller to switch between an AWS Classic Load Balancer (CLB) and an AWS Network Load Balancer (NLB) without deleting the Ingress Controller. Previously, the Ingress Controller had to be force replaced, resulting in an expected outage that could last for several minutes.
```

**nitpick:** Looks okay without this change as well.

**Comment by lihongan** [comment]
Maybe we should not mention `resulting in an expected outage that could last for several minutes`, because the new approach also cause outage for several minutes. see my comment in the [NE-865](https://issues.redhat.com/browse/NE-865).

[Hongan Li](https://issues.redhat.com/secure/ViewProfile.jspa?name=rhn-support-hongli) added a comment - [2022/07/11 4:32 PM](https://issues.redhat.com/browse/NE-865?focusedCommentId=20605886&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20605886)
One more issue should be notified here is that disruptive of routing access is observed after switching from CLB to NLB. Looks the DNS records of the AWS ELB is changed and it needs some time to complete the DNS propagation, so we can see some operators are degraded during the time (about 5 minutes), see example below ...

@miheer please help confirm. thanks

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (jldohmann) [reply release_notes/ocp-4-12-release-notes.adoc]: thanks Gaurav! I ended up chopping this sentence 😄

**Comment by lihongan** [comment]
/lgtm
Thank you!

**Pull request merged**